src/bot/warden/warden.py

```diff
@@ -39,8 +39,13 @@ def check(self, current: DateTime) -> None:
 
 
 class RuleSleepTimeMonTue(IRule):
-    # To be implemented
-    def check(self, current: DateTime) -> None: ...
+    FORBIDDEN_TIME = TimeRange(time(23), time(8))
+    FORBIDDEN_DAYS: tuple[WeekDay, ...] = WORKING_DAYS[:2]
```

Comment:
I offer to use just two days here FORBIDDEN_DAYS: tuple[WeekDay, ...] = WeekDay.Mon, WeekDay.Tue

Also, I realized there was an error. This rule should be applied from Monday to Thursday. Could you please make this change? I believe it would be best to define a new tuple instead of reusing the working_days variable